app-sso/service-operators/issuer-uri-and-tls.hbs.md

```diff
@@ -104,6 +104,8 @@ spec:
 
 Confirm that your `AuthServer` serves a TLS certificate from the specified issuer by visiting its `{.status.issuerURI}`.
 
+> 👉 Learn more about [different TLS Scenarios](./tls-scenario-guides.hbs.md).
```

Comment:
**[question]** is that the required language?

The word "Scenarios" is not super clear to me. I'm thinking something along the lines of _Learn more about different ways to configure TLS for your AuthServer_ or something similar?
Edit: after reading the other page, maybe Scenario is the right word

This link should probably feature more prominently at the top of the page ; and with a bit more explanation "there are many use-cases with certificates, e.g. certs managed outside the cluster, wildcards, etc".